lib/log.js

```diff
@@ -167,6 +215,9 @@ Lug.prototype.activityEvent = function (data) {
 
   this.logger.info('activityEvent', data)
   this.statsd.write(data)
+  // XXX TODO: need to avoid sending the event twice
+  // if it's both an activityEvent and a flowEvent...
+  //sendToAmplitude(data)
```

Comment:
Indeed; I don't think there are any events we want to send that aren't routed through that metrics/events code, so this will probably simplify things.